src/Components/Publishing/ToolTip/__tests__/LinkWithTooltip.test.tsx

```diff
@@ -15,9 +15,18 @@ import {
 import { ToolTip } from "../ToolTip"
 
 describe("LinkWithTooltip", () => {
-  const getWrapper = (context, props) => {
-    const { activeToolTip, tooltipsData, onTriggerToolTip } = context
-    const { text, url, tracking } = props
+  const context = {
```

Comment:
There was a duplicate version of this test file that was more recently updated but also JSX, not sure why, but updated this test and added some typings.

Reply:
Nice 👌